Dockerfile

```diff
@@ -1,9 +1,17 @@
 FROM python:3.8-slim-buster
 
+# Install Java
+RUN apt-get update && \
+    apt-get install -y openjdk-11-jdk && \
+    rm -rf /var/lib/apt/lists/*
+
 WORKDIR /annotation-tools
 
 COPY requirements.txt requirements.txt
 RUN pip3 install -r requirements.txt
 
 COPY . .
 # NOTE: annotation_suite_wrapper.sh won't work without annotation.jar (https://github.com/genome-nexus/genome-nexus-annotation-pipeline) that is not included in the container.
+# Needs a truststore with certificate.
+
+ENTRYPOINT ["bash", "annotation_suite_wrapper.sh"]
```

Comment:
We should change this to a CMD directive since we want the annotation-tools docker image to continue supporting vcf2maf.py and maf2maf.py scripts.